silkworm/db/access_layer.cpp

```diff
@@ -37,6 +37,35 @@ std::optional<BlockHeader> read_header(lmdb::Transaction& txn, uint64_t block_nu
     return header;
 }
 
+// TG ReadTransactions
+static std::vector<Transaction> read_transactions(lmdb::Transaction& txn, uint64_t base_id, uint64_t amount) {
```

Comment:
As `amount` refers to the number of records to load I'd change the name to `count`

Reply:
> As `amount` refers to the number of records to load I'd change the name to `count`

done